### ISSUE TITLE
🐛 `AlreadyAppliedButton` should not be a button and lacks context

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#5611]
## 🐛 Bug

A `button` element with the `disabled` attribute that reads **You have already applied to this.** lacks context (what is the **this** that it is referring to?) and should not be a button. A button does not need punctuation and should convey an action, which this copy does not.

## 🦋 Expected Behaviour

This button should be a non-interactive element and the copy changed to identify what _this_ refers to.

## 📋 Steps to Reproduce

1. View an advertisement that you have already applied to
2. Observe the "Disabled button when a user already applied to a pool opportunity"

## 📸 Screenshots

![Screen Shot 2023-02-03 at 12 39 50](https://user-images.githubusercontent.com/3046459/216716260-73c782d7-b682-4eba-bfd3-d2296b5268c6.png)
![Screen Shot 2023-02-03 at 12 37 34](https://user-images.githubusercontent.com/3046459/216716264-0eaee1a9-f724-430d-9748-62c0dc6c90f3.png)

## ✅ Acceptance Criteria

- [ ] copy reads "You have already applied to this opportunity." (and translate to FR)
- [ ] button is replaced by a non-interactive element

**Comment by petertgiles** [comment]
@Jerryescandon how do you feel about the proposed solution?  Final copy to use?

**Comment by brindasasi** [comment]
@GCTC-NTGC/designers could advise on this ?

**Comment by substrae** [comment]
@brindasasi @petertgiles 

A non-interactive element to replace the button is the right way to go here. Text-wise, I think what @mnigh suggested is fine, barring the lack of centralized terms for pools 😂

> "You have already applied to this opportunity."

**Comment by tristan-orourke** [comment]
@tristan-orourke Roll this into Job Poster update

**Comment by tristan-orourke** [comment]
Rolled into #6318
